### ISSUE TITLE
"No such file or directory"

### ISSUE THREAD

**Issue by deadby16** [lanylow/rbxfpsunlocker-osx#164]
<img width="569" alt="Screen Shot 2022-11-26 at 9 00 18 PM" src="https://user-images.githubusercontent.com/119145555/204117309-5c68f229-c7c9-44c5-b3df-dee217c08179.png">
can someone help i have no idea what im doing wrong 😭 monterey btw !!

**Comment by SeizureSaladd** [comment]
do you even have roblox installed

**Comment by deadby16** [comment]
YES LMFAOOO. i swear im not that dumb😭 i thought that was the problem so i reinstalled it and everything

**Comment by deadby16** [comment]
> do you even have roblox installed

YES LMFAOOO. i swear im not that dumb😭 i thought that was the problem too, so i reinstalled it and everything and it continues to say that

**Comment by lanylow** [comment]
Run these two commands:
`/usr/bin/codesign --remove-signature ~/Applications/Roblox.app/Contents/MacOS/RobloxPlayer`
`/usr/bin/codesign --remove-signature ~/Applications/Roblox.app/Contents/MacOS/Roblox.app/Contents/MacOS/Roblox`

**Comment by deadby16** [comment]
> Run these two commands: `/usr/bin/codesign --remove-signature ~/Applications/Roblox.app/Contents/MacOS/RobloxPlayer` `/usr/bin/codesign --remove-signature ~/Applications/Roblox.app/Contents/MacOS/Roblox.app/Contents/MacOS/Roblox`

<img width="918" alt="Screen Shot 2022-11-27 at 11 49 18 AM" src="https://user-images.githubusercontent.com/119145555/204151820-d9fc0440-9f61-4ae0-b576-5c87022bfa66.png">
<img width="918" alt="Screen Shot 2022-11-27 at 11 52 28 AM" src="https://user-images.githubusercontent.com/119145555/204151826-faff38de-5408-4af2-a1b3-8e2f3d3df678.png">
<img width="868" alt="Screen Shot 2022-11-27 at 11 54 24 AM" src="https://user-images.githubusercontent.com/119145555/204151829-fba66a85-8910-4324-919d-b5930ef527ca.png">

basically, i tried it 3 different ways and none of them worked

**Comment by deadby16** [comment]
> /usr/bin/codesign --remove-signature ~/Applications/Roblox.app/Contents/MacOS/RobloxPlayer

also i did this and it literally keeps saying no such fire or directory omfg 😭

**Comment by lanylow** [comment]
No clue where your roblox is installed then, you have to find where it is or I can't help you.
